### PR TITLE
@GW-4043 - Footer styles

### PR DESCRIPTION
**Pull request by adamccfc**

### CODE REVIEW

File: app/views/govwales/_layout_header.html.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <header id="wg_header" class="wg_header">
   <div class="header" id="header">
-    <div class="header__components">
+    <div class="header__components container">
 
       <div class="header__components__brand" id="wg_brand">
         <a href="https://gov.wales/" title="Welsh Government" class="header__brand" id="wg_logo">
```
Comment (markheathy) [inline app/views/govwales/_layout_header.html.erb]: Need a translator here for Welsh Government title. <%= sanitize(t('header.logo.title')) %>

Comment (markheathy) [reply app/views/govwales/_layout_header.html.erb]: And the URL

File: config/locales/cy.yml [hunk 2]
```diff
@@ -567,3 +567,16 @@ cy:
           - text: 'If you need urgent help and you’re in Northern Ireland call Lifeline on <a href="tel:+448088088000" class="govuk-link">0808 808 8000</a>'
           - text: 'If you need urgent help and you’re in Wales call <a href="tel:+44800132737" class="govuk-link">0800 132 737</a> or text ‘help’ to 81066'
           - text: "If it’s an emergency, call 999"
+  footer:
+    links: |
+      <li class="menu__item"><a href="https://llyw.cymru/cysylltu-ni">Cysylltu â ni</a></li>
```
Comment (nicrodgers) [inline config/locales/cy.yml]: We should minimise the amount of HTML in the locale files.

For having a separate href value for each text link, you can see how this is being done already in the same file:
```
items:
          - text: "Find information about your rights as a tenant"
            href: "https://www.gov.uk/government/publications/covid-19-and-renting-guidance-for-landlords-tenants-and-local-authorities"
```

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en" class="govuk-template">
+<html lang="en">
   <head>
     <meta charset="utf-8" />
     <title><%= t('coronavirus_form.errors.page_title_prefix') if flash[:validation]&.any? %><%= yield :title %> - GOV.UK</title>
```
Comment (markheathy) [inline app/views/layouts/application.html.erb]: We need a task to adjust the header meta asap.

**Review by markheathy — CHANGES_REQUESTED** [review]

File: app/views/govwales/_layout_header.html.erb (hunk 1 above)
Comment (adamccfc) [reply app/views/govwales/_layout_header.html.erb]: I'll pick these up in a separate branch @markheathy

File: app/views/layouts/application.html.erb (hunk 3 above)
Comment (adamccfc) [reply app/views/layouts/application.html.erb]: I've created subtask 4050 to do this @markheathy

File: config/locales/cy.yml (hunk 2 above)
Comment (adamccfc) [reply config/locales/cy.yml]: This has been done now @nicrodgers

File: config/locales/en.yml [hunk 4]
```diff
@@ -100,6 +100,25 @@ en:
...
+      - text: "Copyright statement"
+        href: "https://gov.wales/copyright-statement"
+      - text: "Cookies"
+        href: "https://gov.wales/help/cookies"
+      - text: "Privacy"
+        href: "https://gov.wales/website-privacy-policy"
+      - text: "Terms and conditions"
+        href: "https://gov.wales/terms-and-conditions"
+    logo: |
+      <a href="https://gov.wales/" class="footer__logo" id="footerlogo">
```
Comment (nicrodgers) [inline config/locales/en.yml]: this markup needs removing too

File: app/views/layouts/application.html.erb (hunk 3 above)
Comment (markheathy) [reply app/views/layouts/application.html.erb]: 👍

File: app/views/govwales/_layout_header.html.erb (hunk 1 above)
Comment (markheathy) [reply app/views/govwales/_layout_header.html.erb]: 👍

File: config/locales/en.yml (hunk 4 above)
Comment (adamccfc) [reply config/locales/en.yml]: This is now done @nicrodgers

**Pull request merged**